config/environment.rb

```diff
@@ -3,3 +3,7 @@
 
 # Initialize the Rails application.
 C2::Application.initialize!
+
+C2::Application.configure do
+  config.logger = Logger.new(STDERR)
```

Comment:
it seems that all logs would go to STDERR. This does not seem to unusual. Heroku automatically redirects all Rails logs to STDOUT http://stackoverflow.com/a/7035237/1019369